Code/GraphMol/FMCS/Wrap/testFMCS.py

```diff
@@ -690,6 +691,33 @@ def test11Github2034(self, **kwargs):
         self.assertEqual(mcs.numAtoms, 3)
         self.assertEqual(mcs.numBonds, 3)
 
+    def test19MCS3d(self, **kwargs):
+        random_seed = 0xf00a
+        m1 = Chem.MolFromSmiles('C1CCOC[C@H]1O')
+        m2 = Chem.MolFromSmiles('C1CCOC[C@@H]1O')
+        scaffold = Chem.AddHs(Chem.MolFromSmiles('C1CCOCC1'))
+        AllChem.EmbedMolecule(scaffold, randomSeed=random_seed)
```

Comment:
you could be more explicit and remove the DistGeomHelpers dependency here by just including the mol blocks with the 3D coordinates you're interested in directly here.

By doing the embedding here you introduce the possibility that changes to the conformation generation code may cause these tests to fail.

Reply:
Good point. I thought some more about this, and there were two unit tests that were really redundant, so I kept the one that tests a saved SDF.

Reply:
I've replaced the embedding with mol blocks - thanks!